src/localplayer.cpp

```diff
@@ -169,6 +169,11 @@ bool LocalPlayer::updateSneakNode(Map *map, const v3f &position,
 void LocalPlayer::move(f32 dtime, Environment *env, f32 pos_max_d,
 		std::vector<CollisionInfo> *collision_info)
 {
+	if (!collision_info || collision_info->empty()) {
+		// Node below the feet, update each ClientEnvironment::step()
+		m_standing_node = floatToInt(m_position, BS) - v3s16(0, 1, 0);
```

Comment:
I'm guessing this calculation has to be here insted of in `getStandingNodePos()`?
I'm concerned that this value now depends on the condition, could that cause issues for other code that calls `getStandingNodePos()`?

Reply:
It's primarily here to not break the behaviour for the old sneak code. The standing node position could get out of date in each step, so updating the position here seems best.

Reply:
Ok.